modules/js-and-css/speculation-rules/hooks.php

```diff
@@ -19,9 +19,24 @@ function plsr_print_speculation_rules() {
 		return;
 	}
 
+	// This workaround is needed for WP 6.4. See <https://core.trac.wordpress.org/ticket/56313>.
+	$needs_html5_workaround = (
+		! current_theme_supports( 'html5', 'script' ) &&
+		version_compare( get_bloginfo( 'version' ), '6.4', '>=' ) &&
+		version_compare( get_bloginfo( 'version' ), '6.5', '<' )
```

Comment:
Guard against `-alpha`, `-beta`, and `-RC` from messing up comparisons:

```suggestion
		version_compare( strtok( get_bloginfo( 'version' ), '-' ), '6.4', '>=' ) &&
		version_compare( strtok( get_bloginfo( 'version' ), '-' ), '6.5', '<' )
```

Reply:
```php
var_dump( version_compare( '6.5-beta', '6.5', '<' ) ); // => true ❌
var_dump( version_compare( strtok( '6.5-beta', '-' ), '6.5', '<' ) ); // => false ✅
```